pkg/openstack/rabbitmq.go

```diff
@@ -195,15 +196,28 @@ func reconcileRabbitMQ(
 
 	hostname := fmt.Sprintf("%s.%s.svc", name, instance.Namespace)
 	tlsCert := ""
+	commonName := fmt.Sprintf("%s.%s", hostname, ClusterInternalDomain)
 
 	if instance.Spec.TLS.PodLevel.Enabled {
 		certRequest := certmanager.CertificateRequest{
 			IssuerName: instance.GetInternalIssuer(),
 			CertName:   fmt.Sprintf("%s-svc", rabbitmq.Name),
+			CommonName: &commonName,
 			Hostnames: []string{
 				hostname,
 				fmt.Sprintf("%s.%s", hostname, ClusterInternalDomain),
 			},
+			Subject: &certmgrv1.X509Subject{
+				Organizations: []string{fmt.Sprintf("%s.cluster.local", rabbitmq.Namespace)},
```

Comment:
`cluster.local` can be replaced with the const `ClusterInternalDomain`, like used above in L199

Reply:
ack, done